akka-docs/src/main/paradox/stream/operators/Source/cycle.md

```diff
@@ -14,10 +14,10 @@ Stream iterator in cycled manner.
 
 ## Description
 
-Stream iterator in cycled manner. Internally new iterator is being created to cycle the one provided via argument meaning
-when original iterator runs out of elements process will start all over again from the beginning of the iterator
-provided by the evaluation of provided parameter. If method argument provides empty iterator stream will be terminated with
-exception.
+Stream iterator in cycled manner. Internally a new iterator is being created to cycle the one provided via argument meaning
+when the original iterator runs out of elements process will start all over again from the beginning of the iterator
```

Comment:
out of elements **to** process **it** will start

Reply:
```suggestion
when the original iterator runs out of elements to process it will start all over again from the beginning of the iterator
```